### ISSUE TITLE
Unable to select sample code in docs

### ISSUE THREAD

**Issue by joshkel** [jquense/react-widgets#374]
On some of the docs pages (such as [Getting Started](http://jquense.github.io/react-widgets/docs/#/getting-started)), selecting example code for copy and paste doesn't work.

Observed in both Firefox and Chrome. I think it's caused by the event handlers that CodeMirror installs, but I didn't get very far in figuring out the specifics.

**Comment by zjjott** [comment]
+1 
WTF....

**Comment by jquense** [comment]
is it not allowing copy paste or is the text selection not visible? seems like an odd thing to happen...

**Comment by joshkel** [comment]
Upon further investigation, it looks like selection is occurring but is not visible.

component-playground + CodeMirror implements text selection as one or more absolutely positioned divs with z-index 1 to give the selection background color, while the lines of `<pre>` code are at z-index 2, with all of this inside a `.CodeMirror` div that provides the overall background color. This only works if the code text has a transparent background, but oceanic.css gives a background to `.cm-s-oceanicnext pre` (z-index 2, hiding the selection at z-index 1).

So it's apparently a bug in oceanic.css (at least, a bug in oceanic.css as it's used with component-playground + CodeMirror).

**Comment by jquense** [comment]
hey thanks for tracking that down! pr?

**Comment by joshkel** [comment]
I really don't know that I understand the component-playground code well enough to submit a PR. (It looks like oceanic.css is either automatically generated and/or from a third party? I looked at its comments and git logs but don't see where or how. And I'm not sure how and why your component-playground fork differs from upstream.)

The easiest fix would be to just delete [this line](https://github.com/jquense/component-playground/blob/44663a80d585e82d60fdf8324952e2bdf11e129d/src/themes/oceanic.css#L64), but I don't know if that would have unintended results.

**Comment by jquense** [comment]
its not generated just, vendored in. I would raise the z-index of the selection select tho instead of removing the theme ones

**Comment by joshkel** [comment]
I don't think there's a way to fix things just with z-index. The DOM structure looks like this.

```
<div class="CodeMirror-lines">
  <div style="position: relative; z-index: 1">
    <div class="CodeMirror-selected" style="position: absolute; ..."></div>
  </div>
  <div class="CodeMirror-code">
    <pre class="CodeMirror-line">...</pre>
    <pre class="CodeMirror-line">...</pre>
    <pre class="CodeMirror-line">...</pre>
  </div>
</div>
```

As I understand it, the `"position: relative; z-index: 1"` container and `pre` lines are separate stacking contexts, but as long as `pre` gets a background itself, the selection lines would have to slide in between the `pre`s and their contents (which won't work with separate stacking contexts).

**Comment by jquense** [comment]
ah ok then I think we just need a better selector for the background there prehaps.

**Comment by joshkel** [comment]
I just submitted a PR to fix this; see jquense/component-playground#1.

**Comment by dmk255** [comment]
Looks like this is still an issue.
